### PR TITLE
revert azure experiments

### PR DESCRIPTION
**Pull request by dgoodwin**
- Revert "Add install splay to stagger installs and add to Azure upgrade pre-step."
- Stop gathering all 443 traffic on Azure upgrade jobs.

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml [hunk 1]
```diff
@@ -40,10 +40,6 @@ tests:
     env:
       TEST_REQUIRES_SSH: "yes"
       TEST_TYPE: upgrade-conformance
-    pre:
-    - ref: ipi-install-hosted-loki
```
Comment (deads2k) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:44]: why is loki going away?

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml [hunk 2]
```diff
@@ -40,10 +40,6 @@ tests:
...
-    - ref: tcpdump-service
-    - chain: ipi-azure-pre-stableinitial
```
Comment (deads2k) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:46]: why is the install gone?

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml (hunk 1 above)
Comment (dgoodwin) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:44]: it's part of the default workflow, we overrode pre when we added this and copied in, now we should fall back to the normal workflow which already includes this as our pre section is going away.

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml (hunk 2 above)
Comment (dgoodwin) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml:46]: see above. note that both of these were added when the change went in originally

**Pull request merged**